packages/blade/docs/tokens/Motion.stories.mdx

```diff
@@ -0,0 +1,186 @@
+      <table>
+        <tbody>
+          {Object.entries(theme.motion.delay).map(([key, value]) => (
+            <tr key={key}>
+              <td>{`theme.motion.delay.${key}`}</td>
+              <td>{value}</td>
+            </tr>
+          ))}
+        </tbody>
+      </table>
+      <h3>Duration (in milliseconds)</h3>
+      <table>
+        <tbody>
+          {Object.entries(theme.motion.duration).map(([key, value]) => (
+            <tr key={key}>
+              <td>{`theme.motion.duration.${key}`}</td>
+              <td>{value}</td>
+            </tr>
+          ))}
+        </tbody>
+      </table>
+      <h3>Easing</h3>
+      <h4>standard</h4>
+      <table>
+        <tbody>
+          {Object.entries(theme.motion.easing.standard).map(([key, value]) => (
+            <tr key={key}>
+              <td>{`theme.motion.easing.standard.${key}`}</td>
+              <td>{value}</td>
+              <td style={{ width: '150px' }}>
+                <MovingDiv theme={theme} easing={theme.motion.easing.standard[key]}></MovingDiv>
+              </td>
+            </tr>
+          ))}
+        </tbody>
+      </table>
+      <h4>entrance</h4>
+      <table>
+        <tbody>
+          {Object.entries(theme.motion.easing.entrance).map(([key, value]) => (
+            <tr key={key}>
+              <td>{`theme.motion.easing.entrance.${key}`}</td>
+              <td>{value}</td>
+              <td style={{ width: '150px' }}>
+                <MovingDiv theme={theme} easing={theme.motion.easing.entrance[key]}></MovingDiv>
+              </td>
+            </tr>
+          ))}
+        </tbody>
+      </table>
+      <h4>exit</h4>
+      <table>
+        <tbody>
+          {Object.entries(theme.motion.easing.exit).map(([key, value]) => (
+            <tr key={key}>
+              <td>{`theme.motion.easing.exit.${key}`}</td>
+              <td>{value}</td>
+              <td style={{ width: '150px' }}>
+                <MovingDiv theme={theme} easing={theme.motion.easing.exit[key]}></MovingDiv>
+              </td>
+            </tr>
+          ))}
+        </tbody>
+      </table>
+    </>
+  );
+};
+
+export const MotionExample = () => {
+  const { theme } = useTheme();
+  const ExampleDiv = styled.div`
+    height: 50px;
+    width: 50px;
+    border-radius: 25px;
+    background-color: ${(props) => props.theme.colors.brand.primary[500]};
+    animation: ${(props) =>
+      `resize ${props.theme.motion.duration.gentle3}ms ${props.theme.motion.easing.standard.effective} infinite`};
+    @keyframes resize {
+      0% {
+        transform: scale(1);
+      }
+      50% {
+        transform: scale(1.5);
+      }
+      100% {
+        transform: scale(1);
+      }
+    }
+  `;
+  return <ExampleDiv theme={theme} easing={theme.motion.easing.standard.effective}></ExampleDiv>;
+};
+
+<Motion />
+
+## Example Usage
+
+A circle that scales up and down with a **duration** of `gentle3` and an **easing** of `standard.effective`
+
+<MotionExample />
```

Comment:
render this after code? the placement seems bit off when I look at the documentation as a consumer.